docs/src/pages/reference/cli-reference.md

```diff
@@ -19,6 +19,10 @@ Specifies should port to run on. Defaults to `3000`.
 
 Builds your site for production.
 
+### `astro preview`
+
+Start a local static file server to serve your built `dist/` directory. Useful for previewing your static build before deploying it.
```

Comment:
Should we have a note in here that this should not be used as a production server? In past projects I've seen issues where people use features like this in their hosted environment.